src/home/components/Notification/Notification.tsx

```diff
@@ -52,25 +53,43 @@ export const Notification = () => {
   }
 
   if (currentIndex === 0) {
-    InfiniteSlideHandler(Dummy.length);
+    InfiniteSlideHandler(announcements.length);
```

Comment:
`currentIndex` 상태가 변경되면서 사이드 이펙트로 무한 슬라이딩이 발생하는 것이니 `useEffect()`로 만들면 어떨까요? 

```suggestion
  useEffect(() => {
    const handleInfiniteSlide = (nextIndex: number) => {
      setTimeout(() => {
        if (slideRef.current) {
          setActiveTransition(false);
        }
        setCurrentIndex(nextIndex);
        setTimeout(() => {
          if (slideRef.current) {
            setActiveTransition(true);
          }
        }, 100);
      }, 500);
    };

    if (currentIndex === notificationArray.length - 1) {
      handleInfiniteSlide(0);
    }
  }, [currentIndex, notificationArray.length]);
```

Reply:
적극반영!!!
사실 `useInterval`에서 useEffect 사용하길래 무한 슬라이딩에는 안 써도 되는줄 알았는데... 생각해보니까 서로 다른 역할이더라구요 ㅎㅎㅎ........ 머쓱

[refactor: useEffect를 사용하여 무한 슬라이딩 로직 관리
](https://github.com/yourssu/Soomsil-Web/pull/205/commits/71a071868cc7c3f090fd5296a66833d7d008a6cd)